### PR TITLE
fix(website): setup responsive friendly structure

### PR DESCRIPTION
**Pull request by richbachman**
- [x] Setup more responsive site wrapper so top bar is more responsive friendly. (Not perfect, but better).
- [x] Add `100vh` and `100vw` to style props and proptypes for height and width

Screen capture showing responsive top bar: https://share.getcloudapp.com/E0uz5pNy

### CODE REVIEW

File: packages/paste-website/src/components/site-wrapper/SiteBody.tsx [hunk 1]
```diff
@@ -88,12 +81,14 @@ export const SiteBody: React.FC = ({children}) => {
...
         <Sidebar />
-        <SiteHeader />
-        <SiteMain id="site-main">
-          <ScrollAnchorIntoView />
-          <SiteMainInner>{children}</SiteMainInner>
-          <SiteFooter />
-        </SiteMain>
+        <Box height="100vh" overflow="auto">
+          <SiteHeader />
+          <SiteMain id="site-main">
```
[the remark this answers is not in the document]
Comment (richbachman) [reply packages/paste-website/src/components/site-wrapper/SiteBody.tsx]: I added the `tabIndex={-1}` and `outline:none`. I kept the role on the `main` a couple line below: https://github.com/twilio-labs/paste/pull/474/commits/b428027fb4446d9907c5125e8a6b5e664ae41fee#diff-4dc4c1752543f22e257ce50d14f0a63eL86

I also removed a couple styled divs.

File: packages/paste-website/src/components/site-wrapper/SiteHeader.tsx [hunk 2]
```diff
@@ -26,31 +22,39 @@ export const SiteHeader: React.FC<{}> = () => {
...
+              marginBottom={['space40', 'space40', 'space40', 'space0']}
+            >
+              <Search />
+            </Box>
+            <ContactUsMenu />
+            <Stack orientation="horizontal" spacing="space30">
+              <span>v{version}</span>
+              <Anchor href="https://www.github.com/twilio-labs/paste">
+                <GithubIcon
+                  css={{height: theme.heights.sizeIcon30, width: theme.heights.sizeIcon30}}
```
[the remark this answers is not in the document]
Comment (richbachman) [reply packages/paste-website/src/components/site-wrapper/SiteHeader.tsx]: Changed this to use a style icon.

File: packages/paste-website/src/components/site-wrapper/SiteBody.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (richbachman) [reply packages/paste-website/src/components/site-wrapper/SiteBody.tsx]: Changed to `tabIndex={0}` and moved that to the `main` element along with the `outline: none`.

**Comment by SiTaggart** [comment]
Given that this is unticketed work, and we're now even scope creeping that by attempting to fix something that already not working regarding arrow key page scrolling, this needs to be scoped backed to just fixing the header layout and closed out.

We're just spinning wheels on unplanned work right now. Let's do it properly at the appropriate time.

**Comment by richbachman** [comment]
I've reverted this branch back to before I started changes to the site main content area.

File: packages/paste-website/src/components/site-wrapper/SiteBody.tsx [hunk 3]
```diff
@@ -88,12 +81,14 @@ export const SiteBody: React.FC = ({children}) => {
       )}
       <StyledSiteBody activeTheme={activeTheme}>
         <Sidebar />
-        <SiteHeader />
-        <SiteMain id="site-main">
-          <ScrollAnchorIntoView />
-          <SiteMainInner>{children}</SiteMainInner>
-          <SiteFooter />
-        </SiteMain>
+        <Box height="100vh" overflow="auto">
```
[the remark this answers is not in the document]
Comment (richbachman) [reply packages/paste-website/src/components/site-wrapper/SiteBody.tsx:84]: Postponing until work starts on making it truly responsive.

File: packages/paste-website/src/components/site-wrapper/SiteHeader.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (richbachman) [reply packages/paste-website/src/components/site-wrapper/SiteHeader.tsx:49]: Postponing until work starts on making it truly responsive.

File: packages/paste-website/src/components/site-wrapper/SiteBody.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (richbachman) [reply packages/paste-website/src/components/site-wrapper/SiteBody.tsx:86]: Postponing until work starts on making it truly responsive.

**Pull request merged**